CHANGELOG.md

```diff
@@ -9,13 +9,11 @@ Please see [CONTRIBUTING.md](https://github.com/cucumber/cucumber/blob/master/CO
 ----
 ## [Unreleased] (In Git)
 
-See the [migration guide](./docs/migration.md) for details of how to migrate from 7.x.x to 8.x.x
-
 ### Breaking changes
 
 * Drop support for Node.js 10 and 15, add support for Node.js 16
 * Remove deprecated `--retryTagFilter` option (the correct option is `--retry-tag-filter`)
-* Remove `setDefinitionFunctionWrapper` and step definition option `wrapperOptions`
+* Drop native support for generator functions used as step definitions
```

Comment:
I think this is better worded as "Remove validation that step definition functions are not generators" as I believe that's all that was removed. I don't think we were doing anything else as the user needed to define a step definition wrapper. Similar comment about the wording in the migration guide